.jules/sentinel.md

```
@@ -15,3 +15,8 @@
 **Vulnerability:** `escapeHtml`関数がシングルクォート（'）とダブルクォート（"）をエスケープしていなかったため、属性値などで使用された場合にXSSの潜在的なリスクがあった。
 **Learning:** HTMLエスケープ関数を実装する際は、コンテキスト（本文、属性）に関わらず安全に使えるよう、クォートも含めてエスケープするのが望ましい。
 **Prevention:** 独自のサニタイズ関数を作成せず、定評のあるライブラリを使用するか、OWASP推奨のエスケープ文字セット（&, <, >, ", '）を網羅する。
+
+## 2026-01-18 - [Input Validation for Resource Paths]
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)